site/src/sphinx/server-cors.rst

```diff
@@ -102,3 +102,61 @@ You can also directly add a :api:`CorsPolicy` created by a :api:`CorsPolicyBuild
                                             .build())
                               .newDecorator()));
 
+Configuring CORS via annotation
+-------------------------------
+
+You can also configure CORS for :ref:`server-annotated-service` using the :api:`@CorsDecorator` annotation, e.g.
+
+.. code-block:: java
+
+    import com.linecorp.armeria.common.HttpMethod;
+    import com.linecorp.armeria.common.HttpResponse;
+    import com.linecorp.armeria.common.HttpStatus;
+    import com.linecorp.armeria.server.HttpService;
+    import com.linecorp.armeria.server.ServerBuilder;
```

Comment:
We can omit the import statements which are already explained in the previous examples.